tests/main/interfaces-bluez/task.yaml

```diff
@@ -13,6 +13,11 @@ environment:
     SNAP_NAME: bluez
 
 execute: |
+    if [ "$KARCH" != "x86_64" ]; then
```

Comment:
It'd be nicer to use systems for that instead (see the existing entry above), unless it's too tricky for other reasons. I'm planning on adding globbing support there so it becomes less painful in the future.